src/_includes/layouts/entry/_styles.scss

```diff
@@ -1,3 +1,7 @@
+.entry {
```

Comment:
Let me know if you have a preference on where "entry page specific styles outside of the content" goes! For example `.home` styles are in `pages/home` but I didn't know if following that pattern was right for entries since we have entry templates. I might be misunderstanding the general organization of where different styles should live

Reply:
this is perfect!